index.html

```diff
@@ -6108,11 +6135,126 @@ <h3>Algorithm</h3>
+        <a>JsonLdProcessor</a> functions comprise the data structure known
+        as the <a>internal representation</a>.
+        JSON-LD algorithms are defined against those types.
+      </p>
+
+      <p>
+        Implementations map [[WebIDL]] and [[INFRA]] types to types specific to the
+        programming language in use. For instance:
+      </p>
+
+      <table class="simple">
+        <thead>
+          <tr>
+            <th>WebIDL Type</th>
+            <th>INFRA Type</th>
+            <th>JavaScript</th>
+            <th>Python</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <th><code><a data-cite="WebIDL#idl-USVString">USVString</a></code></th>
+            <th><code><a data-cite="INFRA#string">String</a></code></th>
+            <td><code>String</code></td>
+            <td><code>str</code></td>
+          </tr>
+          <tr>
+            <th><a>JsonLdRecord</a></th>
+            <th><code><a data-cite="INFRA#ordered-map">Map</a></code></th>
+            <td><code>Object</code></td>
+            <td><code>dict</code></td>
+          </tr>
+          <tr>
+            <th><a data-cite="WebIDL##idl-sequence">sequence</a></th>
+            <th><code><a data-cite="INFRA#list">list</a></code></th>
+            <td><code>Array</code></td>
+            <td><code>list</code></td>
+          </tr>
+          <tr>
+            <th><code><a data-cite="WebIDL##idl-long">long</a></code></th>
+            <th><code><a data-cite="XMLSCHEMA11-2#integer">integer</a></code> [[?XMLSCHEMA11-2]]</th>
+            <td><code>Number</code></td>
+            <td><code>int</code></td>
+          </tr>
+          <tr>
+            <th><code><a data-cite="WebIDL##idl-double">double</a></code></th>
+            <th><code><a data-cite="XMLSCHEMA11-2#double">double</a></code> [[?XMLSCHEMA11-2]]</th>
+            <td><code>Number</code></td>
+            <td><code>float</code></td>
+          </tr>
+          <tr>
+            <th><code><a data-cite="WebIDL##idl-boolean">boolean</a></code></th>
+            <th><code><a data-cite="INFRA#boolean">boolean</a></code></th>
+            <td><code>Boolean</code></td>
+            <td><code>bool</code></td>
+          </tr>
+          <tr>
+            <th><code><strong>null</strong></code></th>
+            <th><code><a data-cite="INFRA#nulls">null</a></code></th>
+            <td><code>Null</code></td>
+            <td><code>None</code></td>
+          </tr>
+        </tbody>
+      </table>
+
+      <p>
+        The mappings used for a particular implementation are out of scope for
+        this specification.
+      </p>
+    </section>
+
+    <section id="serialization-and-deserialization">
+      <h4>Serialization and Deserialization</h4>
+    <ins cite="#change_5">
+
+      <p>This section defines the requirements for serializing and deserializing
+        JSON to and from the <a>internal representation</a>, as appropriate,
```

Comment:
```suggestion
        JSON from and to the <a>internal representation</a>, as appropriate,
```
to be consistent with "serializing and deserializing"